pkg/ddevapp/ddevapp.go

```diff
@@ -388,6 +390,22 @@ func (app DdevApp) GetDocroot() string {
 	return app.Docroot
 }
 
+// GetAppRoot return the full path from root to the app directory
+func (app *DdevApp) GetRelativeWorkingDirectory() string {
+	re, err := regexp.Compile(app.AppRoot + `/`)
+
+	if err != nil {
+		return ``
+	}
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		return ``
+	}
```

Comment:
@rfay is this how you would handle errors in such a standard function?

Reply:
No, in almost every case I would return `(string, error)` and return the error. 

There are a few exceptions where it doesn't seem like the error is important (then emit complaint) and at the top-level, sometimes a util.Fail(), which stops with error.

Reply:
I was going to try that initially, will change it.